### PR TITLE
MobileStream.EasyTether version 1.3.4

### PR DESCRIPTION
**Pull request by epic0421**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/63169)

### CODE REVIEW

**Comment by epic0421** [comment]
Not sure why this error is popping up. The only thing that shows up is UAC.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

Looks like we'll need to wait for interactive only installers to be supported so we can add this package to the repository.

File: manifests/m/MobileStream/EasyTether/1.3.4/MobileStream.EasyTether.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: MobileStream.EasyTether
+PackageVersion: 1.3.4
+PackageLocale: en-US
+Publisher: Mobile Stream
+PackageName: EasyTether
+License: Unknown
```
Comment (Trenly) [inline manifests/m/MobileStream/EasyTether/1.3.4/MobileStream.EasyTether.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/m/MobileStream/EasyTether/1.3.4/MobileStream.EasyTether.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: MobileStream.EasyTether
+PackageVersion: 1.3.4
+PackageLocale: en-US
+Publisher: Mobile Stream
+PackageName: EasyTether
+License: Unknown
+Copyright: Copyright (c) Mobile Stream. All rights reserved.
+ShortDescription: EasyTether
```
Comment (Trenly) [inline manifests/m/MobileStream/EasyTether/1.3.4/MobileStream.EasyTether.locale.en-US.yaml:11]: Please provide a better short description

**Review by Trenly — CHANGES_REQUESTED** [review]
Interactive-Only Installer; This package cannot be added due to driver installation prompt
image

File: manifests/m/MobileStream/EasyTether/1.3.4/MobileStream.EasyTether.locale.en-US.yaml (hunk 2 above)
Comment (epic0421) [reply manifests/m/MobileStream/EasyTether/1.3.4/MobileStream.EasyTether.locale.en-US.yaml]: Should be updated now

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/63169#pullrequestreview-1001717821

**Pull request closed without merging**